assets/js/components/notifications/BannerNotifications.js

```diff
@@ -51,6 +51,7 @@ import { READER_REVENUE_MANAGER_MODULE_SLUG } from '../../modules/reader-revenue
 const MODULES_USING_SUBTLE_NOTIFICATIONS = [
 	'ads',
 	READER_REVENUE_MANAGER_MODULE_SLUG,
+	'sign-in-with-google',
```

Comment:
I noticed `READER_REVENUE_MANAGER_MODULE_SLUG` is using a constant, while all other modules have module value hardcoded, not explicitly related to this change, but let's rename this to the `reader-revenue-manager` for consistency. I will also create a new issue to match this across the codebase, we should either have all modules using the constant for the module slug, or RRM modified not to use one, so we have consistency in the approach

Reply:
@zutigrm This change won't add any value here for now and I don't want to wait to run all CI checks again :D We can do this as part of a new PR as part of your new issue. Funnily, in the back end, we use constants for module slugs and in the front end, for many years, we only used constants for module "store names" but not module slugs.

Reply:
Since this issue is basic and almost ready, I think it will go in before 9488. But yes, thanks for flagging this. I already noticed this in my testing too that it is a bit slower to load.